book/01-introduction/1-introduction.asc

```diff
@@ -21,6 +21,6 @@ include::sections/help.asc[]
 
 === Summary
 
-You should have a basic understanding of what Git is and how it's different from the centralized version control system you may have previously been using.
+You should have a basic understanding of what Git is and how it's different from the centralized version control system(s) you may have been using previously. 
```

Comment:
Looks like there's a trailing space here. Also, let's change this to say:

> how it's different from any centralized version control systems you may have been using previously.

Reply:
Dumb question here ... just getting used to GitHub, can I do "git commit --amend" for this on my local system, then push it to my forked repo to the same branch, and redo the pull request? Is that how it works? I'm still reading the chapter on GitHub.